revisions/N0/src/text/dialog_3.asm

```diff
@@ -128,586 +128,1074 @@ Dialog115::
+    db "KLIKK!»@"
+;    db "'BRRING! BRRING!"
+;    db "Hello!  It's me,"
+;    db "Ulrira!  Ask me "
+;    db "anything about  "
+;    db "the island!  If "
+;    db "you get lost,   "
+;    db "give me a call! "
+;    db "Bye! CLICK!'@"
 
 Dialog142::
-    db "'BRRING! BRRING!"
-    db "Hello, this is  "
-    db "Ulrira! ...Well,"
-    db "most Moblins    "
-    db "live in the Mys-"
-    db "terious Forest, "
-    db "but some live in"
-    db "the caves of Tal"
-    db "Tal Heights...  "
-    db "I hope that is  "
-    db "what you wanted "
-    db "to know! CLICK!'@"
+    db "«RRRING! RRRING!"
+    db "Hallo, det er   "
+    db "Ulrik! _… Vel,  "
+    db "de fleste sko-  "
+    db "bolder bor i Den"
+    db "mystiske skogen,"
+    db "men det er også "
+    db "noen i hulene i "
+    db "Tipptopphøydene."
+    db "Håper det var   "
+    db "dette du lurte  "
+    db "på! KLIKK!»@"
+;    db "'BRRING! BRRING!"
+;    db "Hello, this is  "
+;    db "Ulrira! ...Well,"
+;    db "most Moblins    "
+;    db "live in the Mys-"
+;    db "terious Forest, "
+;    db "but some live in"
+;    db "the caves of Tal"
+;    db "Tal Heights...  "
+;    db "I hope that is  "
+;    db "what you wanted "
+;    db "to know! CLICK!'@"
 
 Dialog143::
-    db "'BRRING! BRRING!"
-    db "Yes, this is    "
-    db "Ulrira.  The    "
-    db "Indigestible    "
-    db "Flowers of      "
-    db "Goponga Swamp..."
-    db "Those flowers   "
-    db "are BowWow's    "
-    db "favorite.  Why  "
-    db "don't you take  "
-    db "him for a walk  "
-    db "there?  CLICK!'@"
+    db "«RRRING! RRRING!"
+    db "Ja, du snakker  "
+    db "med Ulrik. De   "
+    db "ufordøyelige    "
+    db "blomstene i Sym-"
+    db "resumpen_… De   "
+    db "blomstene er det"
+    db "beste VovVov vet"
+    db "å spise. Ta ham "
+    db "med på tur dit  "
+    db "da, vel? KLIKK!»@"
+;    db "'BRRING! BRRING!"
+;    db "Yes, this is    "
+;    db "Ulrira.  The    "
+;    db "Indigestible    "
+;    db "Flowers of      "
+;    db "Goponga Swamp..."
+;    db "Those flowers   "
+;    db "are BowWow's    "
+;    db "favorite.  Why  "
+;    db "don't you take  "
+;    db "him for a walk  "
+;    db "there?  CLICK!'@"
 
 Dialog144::
-    db "'BRRING! BRRING!"
-    db "Hi, it's Ulrira!"
-    db "...Have you met "
-    db "everyone on the "
-    db "island?  There's"
-    db "a man named     "
-    db "Richard who     "
-    db "lives in Pothole"
-    db "Field, southeast"
-    db "of the village. "
-    db "Why not pay him "
-    db "a visit?  That's"
-    db "all I can tell  "
-    db "you for now!    "
-    db "Bye!  CLICK!'@"
+    db "«RRRING! RRRING!"
+    db "Hei, det er     "
+    db Ulrik! _… Har du "
+    db "møtt alle på    "
+    db "hele øya? Det er"
+    db "en mann ved navn"
+    db "Rikard som bor  "
+    db "ved Jettegrytene"
+    db "sørøst for      "
+    db "landsbyen. Du   "
+    db "burde stikke in-"
+    db "nom! Det er alt "
+    db "jeg kan fortelle"
+    db "deg nå! "
```

Comment:
```suggestion
    db "deg nå!         "
```
for få mellomrom. eventuelt flytte ha det opp?

Reply:
```suggestion
    db "deg nå! Ha det! "
```